packages/core-bridge/ts/worker-tuner.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @experimental
```

Comment:
nit: move the `@experiemntal` tag line at the end of the comment. Same for other comments in this file.